nemo/collections/asr/parts/jasper.py

```diff
@@ -179,21 +179,58 @@ def forward(self, x):
 
 
 class SqueezeExcite(nn.Module):
-    def __init__(self, channels, reduction_ratio):
+    def __init__(
+        self, channels, reduction_ratio, context_window: int = -1, interpolation_mode='nearest', activation=None
+    ):
+        """
+        Squeeze-and-Excitation sub-module.
+
+        Args:
+            channels: Input number of channels.
+            reduction_ratio: Reduction ratio for "squeeze" layer.
+            context_window: Integer number of timesteps that the context
+                should be computed over, using stride 1 average pooling.
+                If value < 1, then global context is computed.
+            interpolation_mode: Interpolation mode of timestep dimension.
+                Used only if context window is > 1.
+            activation: Intermediate activation function used.
```

Comment:
should this be string or func?

Reply:
Function, ill note that in the docstring.